packages/plugin-core/package.json

```diff
@@ -1301,7 +1292,7 @@
       },
       {
         "command": "dendron.togglePreview",
-        "windows": "windows+ctrl+p",
+        "windows": "ctrl+k v",
```

Comment:
How did we come up with this keybinding? It's very different between mac and windows